i18n.config.ts

```diff
@@ -15,24 +17,48 @@ import pt from "./locales/pt.json";
 import ro from "./locales/ro.json";
 import ru from "./locales/ru.json";
 
+function cleanupEmptyProperties(obj: any, fallback: any): any {
+  Object.keys(obj).forEach((key) => {
+    if (typeof obj[key] === "object") {
+      obj[key] = cleanupEmptyProperties(obj[key], fallback[key]);
+    }
+    if (obj[key] === "") {
+      obj[key] = fallback[key];
+    }
+  });
+  return obj;
+}
```

Comment:
We might want to check if the `obj` and `fallback` params actually are objects, as their types are `any` and thus the type system won't help us.

Not that important though, as the function is only used in this file